packages/utilities/src/utilities/sameheight/sameheight.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Utility that sets elements under a same parent to the sameheight.
+ *
+ * It uses the selector passed to the `sameheight` attribute, to identify
+ * which items should be aligned
+ *
+ * @example
+ * `HTML`
+ *    <div sameheight=".adjustme">
+ *      <elemA class="adjustme">
+ *      <elemB class="adjustme">
+ *    </div>
+ *
+ * `JS`
+ * import {sameheight} from '@carbon/ibmdotcom-utilities';
+ *
+ * sameheight();
+ */
+function sameheight() {
+  /**
+   * Internal function made for avoiding adding more eventlisteners
+   */
+  function setHeight() {
+    if (window.innerWidth > 671) {
+      const sameheightContainers = Array.prototype.slice.call(
+        document.querySelectorAll('[sameheight]')
+      );
+      sameheightContainers.forEach(container => {
+        const selectedElements = Array.prototype.slice.call(
+          container.querySelectorAll(container.attributes['sameheight'].value)
+        );
+        let targetHeight = 0;
+        selectedElements.forEach(elem => {
+          elem.offsetHeight > targetHeight
+            ? (targetHeight = elem.offsetHeight)
+            : false;
+        });
+
+        selectedElements.forEach(elem => {
+          elem.offsetHeight == targetHeight
+            ? (elem.style.height = 'auto')
+            : (elem.style.height = targetHeight + 'px');
+        });
+      });
+    }
+  }
+  setHeight();
```

Comment:
If we move forward with this being a global utility only executed once, then we should probably add some sort of onload event for this. I would still try to steer aware from doing this though. In fact, it might be good to let the component/pattern handle the events too so it can manage the cleanup properly.

Reply:
The logic was changed, as @jeffchew said in some comments up, now it's a normal function.

Reply:
I still think we should still maybe avoid event listeners in this utility. It seems like an easy way to get a memory leak with no way to clean any old events up.

Reply:
Agreed, but let's add some documentation on how an adopter would use it